src/config.js

```diff
@@ -0,0 +1,21 @@
+const hasEnvVariables = typeof process !== 'undefined' && typeof process.env !== 'undefined'
+
+function env(key) {
+  return hasEnvVariables ? process.env[key] : undefined
+}
+
+export const cfg = {
+  ignoreGetDataErrors: env('NODE_ENV') === 'production'
+}
+
+/**
+ * Provides a way to set global configuration options for Data Providers
+ * @param options
+ */
+export function dataProvidersConfig(options) {
+  options = Object(options)
+
+  if ('ignoreGetDataErrors' in options && typeof options.env === 'boolean') {
+    cfg.ignoreGetDataErrors = options.ignoreGetDataErrors
```

Comment:
Why not simply
```
cfg = {...cfg, ...options}
```
?

Reply:
I wanted to be more defensive and not allow users to set some nonsense / harmful code in there. I took the inspiration in the bluebird lib [which has done it similarly](https://github.com/petkaantonov/bluebird/blob/master/src/debuggability.js#L263).

Reply:
Well in such case, I'd expect giving feedback to the user when the param is not accepted. E.g. logging a warning.

Reply:
Fixed, added a warning using `console.warn()`